### PR TITLE
Be a Volunteer 💙

### PR DESCRIPTION
**Pull request by Ptechgithub**
Install and automatic add IP to the domain

### CODE REVIEW

File: volunteer.sh [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+    
+elif [[ $option == "3" ]]; then
+    if systemctl is-active --quiet cfb.service; then
+        read -p "Do you want to add your IP to the worker domain (Set A record)? (yes/no): " create_record_option
+        if [[ $create_record_option == "yes" ]]; then
+            # Retrieve the server's public IPv4 address
+            ipv4_address=$(ifconfig | grep 'inet ' | grep -v '127.0.0.1' | awk '{print $2}')            
+
+            # Cloudflare API variables
+            zone_id="98dc00aea2fe235d9f3b08ec2f5932f2"  # Replace with your actual zone ID
```
Comment (DanielcoderX) [inline volunteer.sh:102]: Also modify this and take these from the user

File: volunteer.sh [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+            api_token="NnO5sXHbIaW03qxZFeBlnOCJH30g9ore_Q0Xu2qm"  # Replace with your actual API token
```
Comment (DanielcoderX) [inline volunteer.sh:103]: Also modify this to get the value from the user

File: volunteer.sh [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+            email="onlycloudflare@gmail.com"  # Replace with your actual email
```
Comment (DanielcoderX) [inline volunteer.sh:104]: this too

File: volunteer.sh [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
+            subdomain="iran"
```
Comment (DanielcoderX) [inline volunteer.sh:105]: again this too

**Review by DanielcoderX — CHANGES_REQUESTED** [review]
Hi, Ptechgithub.
**Thanks** for your interest in contributing and helping us for further improvements.
for some reasons, please build by `trimming path` and getting a `release` also disabling `CGO`.
_Here_ is the fixed build command: `CGO_ENABLED=0 go build -ldflags '-s -w' -trimpath ./relay.go`
Also please fix those that I've mention at the conversation.

**Comment by DanielcoderX** [comment]
Hi again.
Thanks for fixing the build process, but as I mentioned, this part should get fixed and get filled by the user when running the script.

```
            # Cloudflare API variables
            zone_id="98dc00aea2fe235d9f3b08ec2f5932f2"  # Replace with your actual zone ID
            api_token="NnO5sXHbIaW03qxZFeBlnOCJH30g9ore_Q0Xu2qm"  # Replace with your actual API token
            email="onlycloudflare@gmail.com"  # Replace with your actual email
            subdomain="iran"
```

Also there's no need to get the output with name _relay_release_, just a normal **relay** is ok.

**Comment by Ptechgithub** [comment]
If this section is edited, anyone who wants to donate a server must also donate a domain.  But you can choose a default domain and the servers will be behind that domain.  The domain included in this code is iran.ptechfree.eu.org

**Comment by DanielcoderX** [comment]
> If this section is edited, anyone who wants to donate a server must also donate a domain.  But you can choose a default domain and the servers will be behind that domain.  The domain included in this code is iran.ptechfree.eu.org.
> 
> 
> While the server can be added with an A RECORD and there is no need for the user's zone ID and apk token.  You can place a domain and if the number of volunteer servers increases, add some servers to another domain through Cloudflare.

Alright, Let's merge it :)

**Pull request merged**